src/components/TransactionTable.vue

```diff
@@ -69,6 +31,44 @@ export default {
         },
     },
     data() {
+        const columns = [
+            {
+                name: 'hash',
+                label: this.$t('components.tx_hash'),
+                align: 'left',
+            },
+            {
+                name: 'block',
+                label: this.$t('components.block'),
+                align: 'left',
+            },
+            {
+                name: 'date',
+                label: this.$t('components.date'),
+                align: 'left',
+            },
+            {
+                name: 'method',
+                label: this.$t('components.method'),
+                align: 'left',
+            },
+            {
+                name: 'from',
+                label: this.$t('components.from'),
+                align: 'left',
+            },
+            {
+                name: 'to',
+                label: this.$t('components.to_interacted_with'),
+                align: 'left',
+            },
+            {
+                name: 'value',
+                label: this.$t('components.value_transfer'),
+                align: 'left',
+            },
+        ];
+
         return {
             rows: [],
             columns,
```

Comment:
line 144
line 191
line 195

Reply:
I found this that I forgot to transtale
```js
    const tableTitle = computed(() =>
      isTransaction.value ? 'Actions' : 'Latest Transactions'
    );
```

But I couldn't find the three changes that you pointed out. There's only code with no text surrounding those lines. Maybe you refer to another file. Can you check that?